pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -336,36 +336,36 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 	if !foundAny {
 		return nil
 	}
-	klog.Warningf("eviction manager: attempting to reclaim %v", resourceToReclaim)
+	klog.InfoS("eviction manager: attempting to reclaim", "resourceName", resourceToReclaim)
```

Comment:
```suggestion
	klog.InfoS("eviction manager: attempting to reclaim", "resource", resourceToReclaim)
```

resourceName is a bit redundant, might wanna just use resource